update-bootengine

```diff
@@ -34,6 +34,7 @@ DRACUT_ARGS=(
     --omit multipath
     --omit network
     --add iscsi
+    --drivers loop
```

Comment:
```suggestion
    --add-drivers loop
```
The help says `-d, --drivers [LIST]  Specify a space-separated list of kernel modules to exclusively include in the initramfs.` which sounds like we should take `--add-drivers [LIST]  Specify a space-separated list of kernel modules to add to the initramfs.`

Reply:
Did so. Thanks.